assets/src/edit-story/components/panels/design/filter/filter.js

```diff
@@ -32,11 +32,11 @@ import {
   OverlayPreset,
   OverlayType,
 } from '../../../../utils/backgroundOverlay';
-import { Row, ToggleButton, Color, Label } from '../../../form';
+import { Row, ToggleButton, Color } from '../../../form';
```

Comment:
Ah, wait -- shouldn't these use the `ToggleButton` from design-system now?

Reply:
They're temporary anyway only until #6527 is implemented in the very next sprint, so didn't bother.